cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/concepts/GenericConceptBuilder.kt

```diff
@@ -43,6 +43,21 @@ inline fun <reified ConceptClass : Concept> MetadataProvider.newConcept(
         NodeBuilder.log(this)
     }
 
+/**
+ * This function creates a new [Concept] node based on [ConceptClass]. It is neither connected by
+ * the EOG nor the DFG.
+ */
+inline fun <reified ConceptClass : Concept> MetadataProvider.newConceptNoConnect(
```

Comment:
I really dont like this, we should instead have a parameter to `newConcept` like `doNotConnectEOG` or something instead.

Reply:
Added a flag which is mandatory for all builders to prevent that somebody accidentally forgets setting it